cclient/api/tests/cclient_service_init.c

```diff
@@ -32,22 +32,28 @@ static char const *amazon_ca1_pem =
 static char const *const NODE_HOST = "nodes.devnet.iota.org";
 static uint16_t const NODE_PORT = 443;
 
-void cclient_service_setup(iota_client_service_t *const service) {
-  // init service
-  service->http.path = "/";
-  service->http.content_type = "application/json";
-  service->http.accept = "application/json";
-  service->http.host = NODE_HOST;
-  service->http.port = NODE_PORT;
-  service->http.api_version = 1;
-  service->serializer_type = SR_JSON;
-  service->http.ca_pem = amazon_ca1_pem;
+iota_client_service_t *cclient_service_setup() {
```

Comment:
I'd not call this setup for the same reason

Reply:
it's a setup function for unit tests.